packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/shared/base-target-group.ts

```diff
@@ -351,6 +352,21 @@ export abstract class TargetGroupBase extends Construct implements ITargetGroup
 
     return ret;
   }
+
+  protected validateHealthCheck(): string[] {
+    const ret = new Array<string>();
+
+    const intervalSeconds = this.healthCheck.interval?.toSeconds();
+    const timeoutSeconds = this.healthCheck.timeout?.toSeconds();
+
+    if (intervalSeconds && timeoutSeconds) {
```

Comment:
Will this work as expected if either intervalSeconds or timeoutSeconds equals to zero?